internal/console/version_test.go

```diff
@@ -0,0 +1,16 @@
+package console
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestVersionCommand(t *testing.T) {
+	t.Run("Tests if prints current version", func(t *testing.T) {
+		out, err := captureCommandOutput(versionCmd, nil)
+
+		require.NoError(t, err)
+		require.Equal(t, "Keeping Infrastructure as Code Secure v1.1.1\n", out)
```

Comment:
The version string should be converted into a constant (this goes for the actual implementation of the getVersion() as well)

Reply:
Should correct version.go in this PR?

Reply:
Changed to use `currentVersion` constants in version.go and version_test.go